docs/src/docs/asciidoc/user/caching-concepts.adoc

```diff
@@ -37,41 +37,42 @@ and technical decision based upon the requirements and assumptions of your appli
-store, these stores are referred to as tiers.
+You can configure Ehcache to use various data storage areas. 
+When a cache is configured to use more than one storage area, those areas are arranged and managed as tiers - where
+the _hottest_ (most recently accessed) data is kept in the faster (and typically less abundant) tiers, and data that
+is less hot is confined to the slower (and more abundant) tiers.
 
 Data stores supported by Ehcache include:
 
-* On-Heap Store – Utilizes Java's on-heap RAM memory to store cache entries. This tier utilizes the same heap memory as
+* On-Heap Store - Utilizes Java's on-heap RAM memory to store cache entries. This tier utilizes the same heap memory as
 your Java application, all of which must be scanned by the JVM's garbage collector.  The more heap space your JVM
 utilizes, the more your application's performance will be impacted by garbage collection pauses.  This store is
 extremely fast, but is typically your most limited storage resource.
-* Off-Heap Store – Limited in size only by available RAM (tested to as much as 6TB on a single machine!). Not subject
-to Java garbage collection (GC). Is quite fast, yet slower than the On-Heap Store because data must be moved off and
-on the JVM's heap as it is stored and re-accessed.
-* Disk Store – Utilizes a disk (file system) to store cache entries.  This type of storage resource is typically very
+* Off-Heap Store - Limited in size only by available RAM. Not subject
+to Java garbage collection (GC). Is quite fast, yet slower than the On-Heap Store because data must be moved to and
+from the JVM's heap as it is stored and re-accessed.
+* Clustered Store - This data store is a cache on a remote server. The remote server may optionally have a failover
+server providing improved high availability. Since clustered storage comes with performance penalties due to such
+factors as network latency as well as for establishing client/server consistency, this tier, by nature, is slower than
+local off-heap storage.
+* Disk Store - Utilizes a disk (file system) to store cache entries.  This type of storage resource is typically very
 abundant but much slower than the RAM-based stores.
 
 image::EhcacheTerminology.png[]
 
-When a cache is configured to use more than one storage area, those areas are arranged and managed as tiers - where
-the _hottest_ (most recently accessed) data is kept in the faster (and typically less abundant) tiers, and data that
-is less hot remains in the slower (and more abundant) tiers.
-
 == Topology Types
 
 === Standalone
 
-The data set is held in the application node. Any other application nodes are independent with no
-communication between them. If a standalone topology is used where there are multiple application nodes running the
+The data set is held in the application node. If a standalone topology is used where there are multiple application nodes running the
```

Comment:
Since this is being edited, let's move to the recommended one sentence per line format.